core-bundle/contao/dca/tl_member.php

```diff
@@ -206,13 +206,6 @@
 			'eval'                    => array('maxlength'=>64, 'rgxp'=>'phone', 'decodeEntities'=>true, 'feEditable'=>true, 'feViewable'=>true, 'feGroup'=>'contact', 'tl_class'=>'w50'),
 			'sql'                     => "varchar(64) NOT NULL default ''"
 		),
-		'fax' => array
```

Comment:
Why did you remove this field? That's an unnessecary BC break. Data would get lost.

Reply:
I must have committed this unintentionally. Will revert.

Reply:
Reverted in c9ad1f13d3a82d668e890a2ddb2018f473ea8161.